### PR TITLE
Add inline documentation for forest, joint and pairplot

### PR DESCRIPTION
**Pull request by agustinaarroyuelo**
Related to issues  #353,  #352 and #350

### CODE REVIEW

File: arviz/plots/forestplot.py [hunk 1]
```diff
@@ -96,6 +96,24 @@ def plot_forest(
     Returns
     -------
     gridspec : matplotlib GridSpec
+
+    Examples
```
Comment (canyon289) [inline arviz/plots/forestplot.py]: These are great! Are you planning on adding examples for some of the other options/arguments as well?

File: arviz/plots/jointplot.py [hunk 2]
```diff
@@ -56,6 +56,32 @@ def plot_joint(
     axjoin : matplotlib axes, join (central) distribution
     ax_hist_x : matplotlib axes, x (top) distribution
     ax_hist_y : matplotlib axes, y (right) distribution
+
+    Examples
```
Comment (canyon289) [inline arviz/plots/jointplot.py]: Same question for jointplot

File: arviz/plots/forestplot.py (hunk 1 above)
Comment (agustinaarroyuelo) [reply arviz/plots/forestplot.py]: Hi! I just added a forestplot example (before it was only ridgeplot) and one more jointplot.

Comment (canyon289) [reply arviz/plots/forestplot.py]: Sounds good. Feel free to disregard my comment :)

**Pull request merged**